homeassistant/components/blebox/config_flow.py

```diff
@@ -46,6 +52,8 @@ def create_schema(previous_input=None):
         {
             vol.Required(CONF_HOST, default=host): str,
             vol.Required(CONF_PORT, default=port): int,
+            vol.Optional(CONF_USERNAME): str,
+            vol.Optional(CONF_PASSWORD): str,
```

Comment:
```suggestion
            vol.Inclusive(CONF_USERNAME, "auth"): str,
            vol.Inclusive(CONF_PASSWORD, "auth"): str,
```

It is only valid to specify none or both fields.